admin_manual/maintenance/restore.rst

```diff
@@ -30,6 +30,38 @@ install and data folder) to your Nextcloud environment. You could use this comma
 Restore Database
 ----------------
 
+Clean Database before restoring
```

Comment:
From looking at the other headings I assume we should use caps for all words: "Clean Database Before Restoring"